R/xml_interventions_define.R

```diff
@@ -72,60 +70,23 @@ define_vaccine <- defineVaccine
 ##' @param hist If TRUE, then decay is assumed to be step function set to 1 for
 ##'   a year and then to zero for the remainder
 ##' @param resistance Scaling function of insecticide resistance TODO
-##' @examples
-##' VectorInterventionParameters <- list(
-##'   "LLIN" = list(
-##'     deterrency = list(
-##'       decay = list(
-##'         L = "0.7",
-##'         "function" = "weibull"
-##'       ),
-##'       anophelesParams = list(
-##'         "Anopheles gambiae" = list(
-##'           propActive = 1,
-##'           value = "0.5"
-##'         ),
-##'         "Anopheles funestus" = list(
-##'           propActive = 1,
-##'           value = "0.3"
-##'         )
-##'       )
-##'     ),
-##'     preprandialKillingEffect = list(
-##'       decay = list(
-##'         L = "0.4",
-##'         "function" = "weibull"
-##'       ),
-##'       anophelesParams = list(
-##'         "Anopheles gambiae" = list(
-##'           propActive = 1,
-##'           value = "0.6"
-##'         ),
-##'         "Anopheles funestus" = list(
-##'           propActive = 1,
-##'           value = "0.67"
-##'         )
-##'       )
-##'     ),
-##'     postprandialKillingEffect = list(
-##'       decay = list(
-##'         L = "0.45",
-##'         "function" = "weibull"
-##'       ),
-##'       anophelesParams = list(
-##'         "Anopheles gambiae" = list(
-##'           propActive = 1,
-##'           value = "0.3"
-##'         ),
-##'         "Anopheles funestus" = list(
-##'           propActive = 1,
-##'           value = "0.2"
-##'         )
-##'       )
-##'     )
-##'   )
-##' )
-defineVectorControl <- function(baseList, VectorInterventionParameters,
+##' @examples 
```

Comment:
Do NOT break the formatting! This is done for a reason, namely that during the creationg of the manual via roxygen lines longer than 100 characters will be cut off.